AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/README.md

```diff
@@ -236,9 +236,15 @@ type CustomerOrder @model
 }
 
 ```
-3. `amplify push`
+3. If you are using the latest CLI, update cli.json to include `"useExperimentalPipelinedTransformer": false` to ensure that it will use the v1 transformer.
```

Comment:
we might also need to check the value of `transformerversion` flag

Reply:
It worked for me without setting it.